### PR TITLE
Update rules to support eslint 2.x.x

### LINKED ISSUE #23: Upgrade to eslint 2.x (v2.2.0)
http://eslint.org/docs/2.0.0/user-guide/migrating-to-2.0.0

### PR DESCRIPTION
**Pull request by ruiquelhas**
Resolves #23.

### CODE REVIEW

**Comment by rplopes** [comment]
I can confirm that the linting errors I was getting are fixed after applying the diff of this branch :+1:

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ $ npm install eslint babel-eslint eslint-plugin-babel eslint-config-seegno --sav
 
 ## Usage
 
-Create an `.eslintrc` file with the following:
+Create an `.eslintrc.yml` file with the following:
```
Comment (rplopes) [inline README.md]: Why do we need to add an extension?

Comment (ruiquelhas) [inline README.md]: http://eslint.org/docs/user-guide/configuring#configuration-file-formats

> ESLint supports configuration files in several formats:
>
> * YAML - use `.eslintrc.yaml` or `.eslintrc.yml` to define the configuration structure.
> * JSON - use `.eslintrc.json` to define the configuration structure. ESLint's JSON files also allow JavaScript-style comments.
> * Deprecated - use `.eslintrc`, which can be either JSON or YAML.

Comment (rplopes) [inline README.md]: ¯\\\_(ツ)_/¯ ok, if they say so, then +1

**Pull request merged**